src/repley/shadow/preload.cljs

```diff
@@ -0,0 +1,12 @@
+(ns repley.shadow.preload
+  "May be added to `shadow-cljs.edn` `:preloads` to add tap on start."
+  (:require
+   [repley.shadow.remote :as r]
+   goog.object))
+
+(let [port (-> (goog.object/get js/CLOSURE_DEFINES "repley.shadow.build.opts")
+               (js->clj :keywordize-keys true)
+               (get-in [:http :port]))
+      opts (cond-> {:mode "no-cors"}
```

Comment:
I've had to add `no-cors` to make it work on my machine, but perhaps this shouldn't be the default?

At any rate, `repley.shadow.preload` and `repley.shadow.build` may be considered as a blueprint for users to customize further. (Example: I'm using a modified `repley.shadow.build/start-and-listen` in order to add readers for edn.)